tests/testthat/_snaps/validation.md

```diff
@@ -1,4 +1,230 @@
+# bad values
+
+    Code
+      baguette:::validate_args(model = "mars", times = 5L, control = control_bag(),
+      cost = NULL)
+    Condition
+      Error:
+      ! `base_model` must be one of "CART", "C5.0", "MARS", or "nnet", not "mars".
+      i Did you mean "MARS"?
+
+---
+
+    Code
+      baguette:::validate_args(model = "MARS", times = 1, control = control_bag(),
+      cost = NULL)
+    Condition
+      Error:
+      ! `times` must be a whole number larger than or equal to 2, not the number 1.
+
+---
+
+    Code
+      baguette:::validate_args(model = "MARS", times = -1L, control = control_bag(),
+      cost = NULL)
+    Condition
+      Error:
+      ! `times` must be a whole number larger than or equal to 2, not the number -1.
+
+---
+
+    Code
+      baguette:::validate_args(model = "MARS", times = 5L, control = 2, cost = NULL)
+    Condition
+      Error:
+      ! The control object should be a list created by `control_bag()`.
+
+---
+
+    Code
+      bagger(Sepal.Length ~ ., data = iris, times = 2L, base_model = "CART", cost = 2)
+    Condition
+      Error in `bagger()`:
+      ! `cost` can only be a scalar when there are two levels.
+
+# wrong y for C5
+
+    Code
+      bagger(Sepal.Length ~ ., data = iris, times = 2L, base_model = "C5.0")
+    Condition
+      Error in `validate_outcomes_are_factors()`:
```

Comment:
All of our snapshots are going to change once these hardhat functions can take calls